client/components/profilePage/friendRecommendations.jsx

```diff
@@ -1,32 +1,40 @@
-import React, {useEffect} from 'react';
-import { Card, Col, Row } from 'react-bootstrap';
+import React, { useEffect } from 'react';
+import { Card, Col, Row, Accordion, Button } from 'react-bootstrap';
 
-export default function FriendRecommendations({friendsList}) {
+export default function FriendRecommendations({ friendsList }) {
   const reviewList = friendsList;
 
   return (
-
     <div>
       <Row>
-        <Col>
-        <p>What friends are recommending</p>
-        </Col>
+        <div
+          style={{ fontSize: '23px' }}
+          className='userClubsHeader font-weight-bold'
+        >
+          Friends recommendations
+        </div>
       </Row>
-      {reviewList.length > 0 &&
-      reviewList.slice(0,3).map((friend) =>{
-        return (
-          <Row key={friend._id}>
-            <Col>
-              <Card className="recommendation-cards">
-                <p>{friend.username} recommends...</p>
-                <p>{friend.recommendation.summary}</p>
-                <p>{friend.recommendation.description}}</p>
+
+      <Accordion defaultActiveKey='0'>
+        {reviewList.length > 0 &&
+          reviewList.map((friend, i) => {
+            return (
+              <Card key={friend._id}>
+                <Card.Header>
+                  <Accordion.Toggle as={Button} variant='link' eventKey={i + 1}>
```

Comment:
Don't think it really matters, but why is the +1 needed? Does i alone not work?

Reply:
nope, it seems Accordion starts at 1 and not 0